_data/navigation.yml

```diff
@@ -1,43 +1,61 @@
 primary:
-  - text: Welcome
+  - text: About this guide
     href: /
   - text: Discover the current state
-    links:
-      - text: Overview
-        href: /discover/
-      - text: Understand users and their 'as is' process
-        href: /discover/users/
-      - text: Understand your stakeholders
-        href: /discover/stakeholders/
-      - text: Understand the technical landscape
-        href: /discover/technical/
-      - text: Understand the regulatory landscape
-        href: /discover/compliance/
+    href: /discover/
   - text: Define the future state
-    links:
-      - text: Overview
-        href: /future/
+    href: /define/
+  - text: Deliver the "right" way
+    href: /deliver/
+  - text: Enable partners
+    href: /partners/
+
+about:
+    - text: About this guide
+      href: '#about-this-guide'
+
+discover:
+    - text: Discover the current state
+      href: /discover/
+    - text: Understand users and their 'as is' process
+      href: /discover/users/
+    - text: Understand your stakeholders
+      href: /discover/stakeholders/
+    - text: Understand the technical landscape
+      href: /discover/technical/
+    - text: Understand the regulatory landscape
+      href: /discover/compliance/
```

Comment:
It looks like there's some variability in the indentation of the `yaml` list items. It doesn't appear to be breaking anything, but it's probably a good idea to use consistent spacing. But don't tell anyone whether you're using tab versus spaces ;)

Reply:
(I'm pretty sure `YAML` doesn't impose strict indenting rules, but it's good to be consistent for readability and consistency.)